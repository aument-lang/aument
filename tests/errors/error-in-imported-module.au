import "./error-in-imported-module-1.au";
