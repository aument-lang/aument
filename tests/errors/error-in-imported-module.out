interpreter error(1) in -: incompatible values for binary operation
1 | print "A" + 1;
interpreter error(7) in -: came from here
1 | import "./error-in-imported-module-1.au";
